scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/matcher/simplify/IdentityLossReporter.java

```diff
@@ -43,16 +44,16 @@
  * @param <T> - the type that is not being simplified.
  */
 @OpHints(hints = {Simplification.FORBIDDEN})
-@OpClass(names = "lossReporter")
-public class IdentityLossReporter<T> implements LossReporter<T, T>, Op {
+@OpClass(names = "lossReporter", priority= Priority.VERY_HIGH)
+public class IdentityLossReporter<U, T extends U> implements LossReporter<T, U>, Op {
```

Comment:
Why does this op need to be an `OpClass` rather than a more concise `OpField` using a lambda?